examples/binding/abl-imatinib/explicit.yaml

```diff
@@ -3,15 +3,15 @@ options:
   minimize: yes
   verbose: yes
   output_dir: .
-  number_of_iterations: 1000
+  number_of_iterations: 500
```

Comment:
These are local changes, but they shouldn't be a hinderance---they just try to balance simulation and non-simulation time for optimal performance.